packages/mobile/src/icons/VerificationDenied.tsx

```diff
@@ -5,17 +5,17 @@ function VerificationDenied() {
   return (
     <Svg width="60" height="60" viewBox="0 0 60 60" fill="none" xmlns="http://www.w3.org/2000/svg">
       <Circle cx="30" cy="30" r="30" fill="#F9F6F0" />
-      <Circle cx="30" cy="30" r="11.25" stroke="#EA6042" stroke-width="1.5" />
+      <Circle cx="30" cy="30" r="11.25" stroke="#2E3338" stroke-width="1.5" />
       <Path
         d="M34.5 25.5L25.5 34.5"
-        stroke="#EA6042"
+        stroke="#2E3338"
         stroke-width="1.5"
         stroke-linecap="round"
         stroke-linejoin="round"
```

Comment:
These props are not directly understood by `react-native-svg`.
The original SVG should be processed using https://react-svgr.com/playground/?native=true
Making sure the `React Native` option is checked.

Reply:
Didn't know about this, Thanks! 🙏